lib/workers/repository/finalise/index.ts

```diff
@@ -15,4 +17,17 @@ export async function finaliseRepo(
     `Action Required: Fix Renovate Configuration`
   );
   await clearRenovateRefs();
+  const prList = await platform.getPrList();
+  if (
+    prList?.some(
+      (pr) =>
+        pr.state === 'merged' &&
+        pr.title !== 'Configure Renovate' &&
+        pr.title !== config.onboardingPrTitle
+    )
+  ) {
+    logger.debug('Repo is activated');
+    config.repoIsActivated = true;
```

Comment:
@rarkins I think this should use memcache (like the http stats) instead of changing our global config

https://github.com/renovatebot/renovate/blob/637585c5fd9f7d3239318ad9e9e3ec288ec3c6e1/lib/workers/repository/stats.ts#L7

Reply:
At the moment it's inline with the existing config.repoIsOnboarded, so I'd prefer to leave it. The app reads the result/status of each run to determine if a repo is onboarded, and this is intended to let me see if the repo is truly active or they just onboarded and never merged a PR after that.

I agree we should look into a first class stats/metadata gathering concept, not necessarily in memCache but perhaps a module which is reset ever run